app/js/actions/AuthActions.js

```diff
@@ -40,9 +40,13 @@ export function logoutUser() {
 export function loginUser() {
   return dispatch => {
     dispatch(beginAjaxCall());
-    return webAPI(`/login`, 'POST')
+    return webAPI(`/users`, 'GET')
```

Comment:
in order to fetch the current user, you can now get it through `/users/me`. Querying for the user at `/users` would return an array of all the users in the db! I'm sure that's not what you were expecting